### PR TITLE
Highlight operators + improve consistency with Vim syntax

### PR DESCRIPTION
**Pull request by Avi-D-coder**

### CODE REVIEW

File: queries/rust/highlights.scm [hunk 1]
```diff
@@ -97,7 +97,7 @@
 
 (lifetime (identifier) @label)
 
-(self) @variable.builtin
+(self) @keyword
```
Comment (kyazdani42) [inline queries/rust/highlights.scm:100]: i believe self should stay a builtin variable. If you really want this to be a keyword, you could override it in one of your own query files :)

**Review by kyazdani42 — CHANGES_REQUESTED** [review]

File: queries/rust/highlights.scm (hunk 1 above)
Comment (theHamsta) [reply queries/rust/highlights.scm:100]: Or link variable.builtin to keyword. It's mostly `this` and `self`

File: queries/rust/highlights.scm [hunk 2]
```diff
@@ -165,8 +165,42 @@
...
+"-"
+"-="
+"+"
+"+="
+"/"
+"/="
+">"
+"<"
+">="
+"<="
```
Comment (TravonteD) [inline queries/rust/highlights.scm:197]: This is duplicated from line 171
```suggestion
```

File: queries/rust/highlights.scm (hunk 1 above)
Comment (Avi-D-coder) [reply queries/rust/highlights.scm]: Fair enough. I opened #327 in this regard.

**Review by kyazdani42 — APPROVED** [review]
looks good now :) thanks

**Pull request merged**